### DISCUSSION TITLE
please help: download failed, how do i fix?

### DISCUSSION THREAD

**Discussion by Awesome820** [aclap-dev/video-downloadhelper#756] [Help and Q&A]
### Your question

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://watchseriesstream.com/watch-movie/watch-the-bad-guys-full-online-78364.8184337",
  "error": {
    "details": "SideDownload error: : [https @ 0000018a61bed940] Stream ends prematurely at 222444, should be 1194552\r\n[https @ 0000018a61bee580] Stream ends prematurely at 1408356, should be 1561528\r\n[aac_adtstoasc @ 0000018a639ad180] Error parsing ADTS frame header!\r\n[mp4 @ 0000018a63a08000] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input\r\n[aost#0:1/copy @ 0000018a639dc700] Error submitting a packet to the muxer: Invalid data found when processing input\r\n[out#0/mp4 @ 0000018a61bc30c0] Error muxing a packet\r\n",
    "downloadable_id": "downloadable_3309108307671139",
    "error": "coapp_failure",
    "id": "coapp_failure_3aa68cc5-0bbd-4f4b-8fd4-54f9e6290186"
  }
}

### Addon & CoApp version and details

version: 9.0.1.22
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Users\\Jakes\\Desktop\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 2,
    "successfull_dl": 2
}

**Comment by Mattmjs** [comment]
It's works for me on Firefox. I got a few minutes before stopping download as I'm not wanting this. See if the situation is
different on Firefox or you get the same problem. And when exactly does the problem occur ? 
At the beginning of download, part way through or at the end ?

**Reply by Awesome820** [reply]
tried on firefox, now i get a new issue: Download failed, Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error (it's anonymous!).

**Reply by Awesome820** [reply]
also it hapens during the download right near the end
